examples/widgets/formula.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Category DataView Example</title>
+
+  <!-- Custom CSS -->
+  <link rel="stylesheet" type="text/css" href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css" />
+  <link rel="stylesheet" href="https://libs.cartocdn.com/airship-style/v2.3/airship.css" />
+
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    #map {
+      width: 100vw;
+      height: 100vh;
+    }
+
+    .widgets {
+      position: absolute;
+      right: 10px;
+      top: 10px;
+      height: 400px;
+    }
+
+    as-formula-widget {
+      height: auto;
+    }
+  </style>
+</head>
+
+<body>
+  <section id="map"></section>
+
+  <section class="widgets">
+    <as-formula-widget class="as-p--16" heading="Total Countries" description="Total number of countries">
+    </as-formula-widget>
+  </section>
+
+  <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
+  <script src="https://unpkg.com/deck.gl@8.2.0-alpha.2/dist.min.js"></script>
+  <script src="/airship/airship.js"></script>
+
+  <script src="/dist/umd/index.min.js"></script>
+
+  <script>
+    const widgetElement = document.querySelector('as-category-widget');
+
+    async function initialize() {
+      carto.setDefaultCredentials({ username: 'public' });
+
+      const deckMap = carto.viz.createMap();
+
+      const countriesLayer = new carto.viz.Layer('ne_50m_admin_0_countries');
+      await countriesLayer.addTo(deckMap);
+
+      // Formula Widget
+      const formulaDataview = new carto.data.FormulaDataView(countriesLayer, 'cartodb_id', {
```

Comment:
I think we'll have to change it to `carto.dataview.Formula`, after @manmorjim current PR. Also, as we will probably change the 'count', to avoid the need of a param, maybe we can just use 'sum' or another formula.

Regarding to the airship panel, there is right now an example for CategoryDataView I changed a bit, to add the title above the widget. You can probably copy that layout for this, and we'll get a more homogeneous look, and a basic responsive panel

Reply:
I have just pushed the changes.

Regarding what we talked yesterday of renaming `Category` class (or any other that has the same issue) to `CategoryWidget`, I think we should do it as soon as we can. I have had to add another alias for FormulaDataview in the Formula (FormulaWidget) class. We are gonna end up with alias everywhere.